packages/compiler/src/experimental/typekit/kits/enum.ts

```diff
@@ -40,9 +41,18 @@ export interface EnumKit {
   create(desc: EnumDescriptor): Enum;
 
   /**
-   * Build an equivalent enum from the given union. Union variants which are
+   * Build an equivalent enum from the given union.
+   *
+   *
+   * @remarks
+   *
+   * Union variants which are
    * not valid enum members are skipped. You can check if a union is a valid
    * enum with {@link UnionKit.union}'s `isEnumValue`.
+   *
+   * Any API documentation will be rendered and preserved in the resulting enum.
+   * - No other decorators are copied from the union to the enum
```

Comment:
```suggestion
   * No other decorators are copied from the union to the enum.
```

Reply:
I did have that as such, but the hover experience is not great. With that said, I can definitely change that in the next PR i just don't want to dismiss approvals for this 😢